protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -60,9 +69,40 @@ contract Account {
     // The global AppStorage state stores a mapping from account address to Account.State.
     struct State {
         Field field; // A Farmer's Field storage.
-        AssetSilo bean; // A Farmer's Unripe Bean Deposits only as a result of Replant (previously held the V1 Silo Deposits/Withdrawals for Beans).
-        AssetSilo lp;  // A Farmer's Unripe LP Deposits as a result of Replant of BEAN:ETH Uniswap v2 LP Tokens (previously held the V1 Silo Deposits/Withdrawals for BEAN:ETH Uniswap v2 LP Tokens).
-        Silo s; // A Farmer's Silo storage.
+
+        /*
+         * @dev (Silo V1) A Farmer's Unripe Bean Deposits only as a result of Replant
+         *
+         * Previously held the V1 Silo Deposits/Withdrawals for Beans.
+
+         * NOTE: While the Silo V1 format is now deprecated, this storage slot is used for gas
+         * efficiency to store Unripe BEAN deposits. See {FIXME(doc)} for more.
+         */
+        AssetSilo bean; 
+
+        /*
+         * @dev (Silo V1) Unripe LP Deposits as a result of Replant.
+         * 
+         * Previously held the V1 Silo Deposits/Withdrawals for BEAN:ETH Uniswap v2 LP Tokens.
+         * 
+         * BEAN:3CRV and BEAN:LUSD tokens prior to Replant were stored in the Silo V2
+         * format in the next storage slot, `s.a[account].silo`.
+         *
+         * NOTE: While the Silo V1 format is now deprecated, this storage slot is used for gas
+         * efficiency to store Unripe BEAN:3CRV deposits. See {FIXME(doc)} for more.
+         */
+        AssetSilo lp; 
+
+        /*
+         * @dev Silo V2
+         */
+        Silo s;
+        
+        /*
+         * @notice DEPRECATED
+         * 
+         * @dev FIXME(doc)
+         */
         uint32 votedUntil; // DEPRECATED – Replant removed on-chain governance including the ability to vote on BIPs.
         uint32 lastUpdate; // The Season in which the Farmer last updated their Silo.
```

Comment:
if this isn't deprecated we should probably add some spacing between it and `votedUntil`